src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { DEFAULT_IMAGE_TEXT } from 'src/constants/constant';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+
+export class HeaderComponent {
+  imgText = DEFAULT_IMAGE_TEXT;
```

Comment:
fields could be [readonly](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#readonly-and-const).